sdk/servicebus/service-bus/src/session/messageSession.ts

```diff
@@ -176,22 +167,10 @@ export class MessageSession extends LinkEntity {
    * Default: `300 * 1000` (5 minutes);
    */
   maxAutoRenewDurationInMs: number;
-  /**
-   * @property {number} [newMessageWaitTimeoutInMs] The maximum amount of idle time the session
-   * receiver will wait after a message has been received. If no messages are received in that
-   * time frame then the session will be closed.
-   */
-  newMessageWaitTimeoutInMs?: number;
   /**
    * @property {boolean} autoRenewLock Should lock renewal happen automatically.
    */
   autoRenewLock: boolean;
-  /**
-   * @property {SessionCallee} callee Describes who instantiated the MessageSession. Whether it was
-   * called by the SessionManager or it was called standalone.
-   * - Default: "standalone"
-   */
-  callee: SessionCallee;
```

Comment:
Now that we call MessageSession from sessionReceiver, we don't need this?

Reply:
If I recall correctly, the reason we don't need this is because we removed the implementation of `SessionManager`. That is when we had to make the distinction between whether a stand alone session receiver created the `MessageSession` or a session receiver created by the SessionManager did so

Reply:
Ah never mind, I realized what you said means the same. The words "only from sessionReceiver" were missing, so I misunderstood :)